libs/types-common/src/Data/Code.hs

```diff
@@ -95,20 +95,27 @@ instance ToHttpApiData Value where
 newtype Timeout = Timeout
   {timeoutDiffTime :: NominalDiffTime}
   deriving (Eq, Show, Ord, Enum, Num, Fractional, Real, RealFrac)
+  deriving (S.ToSchema) via (Schema Timeout)
+
+instance ToSchema Timeout where
+  schema = Timeout . fromIntegral <$> (roundDiffTime . timeoutDiffTime) .= schema
+    where
+      roundDiffTime :: NominalDiffTime -> Int32
+      roundDiffTime = round
 
 -- | A 'Timeout' is rendered as an integer representing the number of seconds remaining.
 instance ToByteString Timeout where
   builder (Timeout t) = builder (round t :: Int32)
 
 -- | A 'Timeout' is rendered in JSON as an integer representing the
 -- number of seconds remaining.
-instance ToJSON Timeout where
-  toJSON (Timeout t) = toJSON (round t :: Int32)
+instance A.ToJSON Timeout where
+  toJSON (Timeout t) = A.toJSON (round t :: Int32)
```

Comment:
Remove this instance and derive it `via Schema Timeout`.

Reply:
Done in 5413dd1.